kernelci/build.py

```diff
@@ -802,6 +802,28 @@ def _kernel_config_enabled(self, config_name):
         cmd = 'grep -cq CONFIG_{}=y {}'.format(config_name, dot_config)
         return shell_cmd(cmd, True)
 
+    def _kernel_config_getkey(self, opt_name):
+        dot_config = os.path.join(self._output_path, '.config')
+        with open(dot_config, 'r') as fp:
+            lines = fp.readlines()
+            for line in lines:
```

Comment:
This is simpler syntax, more efficient as all the lines don't have to be kept in memory and the loop will stop when the line with the config has been found:
```suggestion
            for line in fp:
```

Reply:
Done